### PR TITLE
report used space to pd

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@disksing @BusyJay PTAL

### CODE REVIEW

**Comment by BusyJay** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -1548,6 +1548,8 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         let mut used_size = flush_engine_properties_and_get_used_size(self.engine.clone());
         used_size += self.snap_mgr.get_total_snap_size();
 
+        stats.set_used_size(used_size);
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: do we need to get the WAL and Info log file size now?

Comment (zhangjinpeng1987) [inline src/raftstore/store/store.rs]: Any way to get current WAL size?

Comment (zhangjinpeng1987) [inline src/raftstore/store/store.rs]: Maybe we should add current memtable size to used size.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**